src/integTest/groovy/net/wasdev/wlp/gradle/plugins/AbstractIntegrationTest.groovy

```diff
@@ -66,16 +51,19 @@ abstract class AbstractIntegrationTest {
         }
     }
 
-    protected static File createFile(File parent) {
-        File destFile = new File(parent, 'build.gradle')
-        File sourceFile = new File("build/resources/integrationTest/build.gradle")
-        if (!sourceFile.exists()){
-            throw new AssertionError("The source file '${sourceFile.canonicalPath}' doesn't exist.")
+    protected static File createTestProject(File parent, File sourceDir) {
+        if (!sourceDir.exists()){
+            throw new AssertionError("The source file '${sourceDir.canonicalPath}' doesn't exist.")
         }
         try {
-            FileUtils.getFileUtils().copyFile(sourceFile, destFile, null, true);
+            AntBuilder ant = new AntBuilder()
+            ant.copy(todir: parent.toString()){
+                fileset(dir: sourceDir.toString())
+            }
         } catch (IOException e) {
-            throw new AssertionError("Unable to create file '${destFile.canonicalPath}'.")
+            throw new AssertionError("Unable to create file '${parent.canonicalPath}'.")
+        } catch (GroovyCastException e) {
```

Comment:
AntBuilder copy will copy the test resource files correctly but throws a cast exception after doing so. I haven't found another work around or a different way to copy directories. Here is the exception plus message: 

`org.codehaus.groovy.runtime.typehandling.GroovyCastException: Cannot cast object 'org.apache.tools.ant.taskdefs.Copy@2c4b5f9c' with class 'org.apache.tools.ant.taskdefs.Copy' to class 'java.io.File'`